src/main/main.ts

```diff
@@ -270,12 +270,13 @@ async function initializeApp() {
 
     if (update) {
       ipcMain.handle("start player update", async () => {
-        await playerUpdate(update, updateOptions);
+        update.player.updateRequired = true;
+        await performUpdate(update, updateOptions);
```

Comment:
How can I modify `updateRequired` in `player` object?

Reply:
Ooh right.
```suggestion
        await performUpdate({ ...update, player: { ...update.player, updateRequired: true } }, updateOptions);
```

It is admittedly verbose, though. We could've used immer but I don't feel we should add a dependency because of this.

If you're sure that there are no references after we call `"start player update"` this isn't a concern.

Reply:
hmm.. I'm concerned too 🤔 
Now we have no references after call `start player update` and `start launcher update` then no problems.
But for the future, I think we could use immer because of this

Reply:
But now done